test/__snapshots__/katex-spec.js.snap

```diff
@@ -1,5 +1,435 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`A font parser \\boldsymbol should inherit mbin/mrel from argument 1`] = `
```

Comment:
This snapshot is surprising hard to read.  It would be cool if we could choose to filter the props to the tree to only show the `value` and `classes` on some tests.

Reply:
Sounds like a good idea.  Maybe for after #1336?

Reply:
The snapshot should be easier to read in future PRs if we end of having to tweak something.